.github/workflows/ci.yaml

```diff
@@ -43,11 +43,11 @@ jobs:
         with:
           go-version-file: "${{ github.workspace }}/go.mod"
       - name: Run golangci-lint
```

Comment:
out of curiosity, why don't you have a golanglint config in this repo?

Reply:
we forgot I think,

also I tried to add one now, but that cause way too many changes, so I will create a follow up for that

Reply:
because without config majority of linters are disabled. it's okey for this PR, but IMO we should add a ticket to port operator's linter config to this repo, with TODO of enable one by one in future.